README.md

```diff
@@ -40,6 +40,7 @@ require 'sidekiq/web'
 require 'sidekiq-statistic'
 
 use Rack::Session::Cookie, secret: 'some unique secret string here'
+Sidekiq::Web.instance_eval { @middleware.reverse! } # Last added, First Run
```

Comment:
https://github.com/mperham/sidekiq/issues/2459
this workaround woks but there might be a better solution!